dev-php/awl/awl-0.58.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit vcs-snapshot eutils
```

Comment:
why is `eutils` still here?

Reply:
leftover from einstalldocs .. will remove it